WooCommerce/Classes/ViewRelated/Products/Media/WordPressMediaLibraryPickerDataSource.swift

```diff
@@ -22,8 +22,7 @@ extension WordPressMediaLibraryPickerDataSource {
         }
 
         func cancelImageRequest(_ requestID: WPMediaRequestID) {
-            // TODO-2073: implement image request cancelling once the image fetching is not tied to `Media` since it needs to keep
-            // track of the task.
+            // No image is shown for the media group.
```

Comment:
Is this still part of https://github.com/woocommerce/woocommerce-ios/issues/2073?

Reply:
great catch! I removed this TODO item from https://github.com/woocommerce/woocommerce-ios/issues/2073